EntranceRando.py

```diff
@@ -313,9 +313,9 @@ def find_pairing(dead_end: bool, require_new_regions: bool) -> Optional[Tuple[En
                 if all(e.connected_region in er_state.placed_regions for e in lookup):
                     return None
 
-            raise Exception(f"None of the available entrances had valid targets.\n"
-                            f"Available exits: {er_state.placeable_exits}\n"
-                            f"Available entrances: {lookup}")
+            raise RuntimeError(f"None of the available exits are valid targets for the available entrances.\n"
```

Comment:
Actually the opposite is true, none of the entrances were valid targets for the exits